articles/users/search/v2/index.md

```diff
@@ -1,6 +1,5 @@
 ---
 description: This page lists several examples of user search queries using query string syntax.
-section: apis
 crews: crew-2
 toc: true
 ---
```

Comment:
Just a note that we do not have a version-warning panel here. Not sure if we want to that's why I'm not asking for the change.